packages/axios-extension/src/abap/adt-catalog/handlers/transport.ts

```diff
@@ -68,9 +69,9 @@ function getTransportList(doc: Document): string[] {
     } else if (locked) {
         return getLockedTransport(doc);
     } else if (LocalPackageText.includes(localPackage)) {
-        return [''];
-    } else {
         return [];
+    } else {
+        throw new Error('Unable to parse ADT response');
```

Comment:
Do we have i18n support already?

Reply:
> > #518
> > Change API of ADT request getTransportRequests to return transport req metadata associated with transport numbers
> 
> @slin-sap, do you mind adding a brief reason for this change? The issues and PRs are good documentation to trace the changes to the code. Thanks

See updated the PR description

Reply:
Apparently no in the entire open source project.